modules/ROOT/pages/rest-sdk/edits/customization-layer.adoc

```diff
@@ -0,0 +1,54 @@
+= Create a Customization Layer
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+After creating your connector, optionally create an instance of your existing
+connector descriptor to generate a new layer of functionality and customization
+to describe other use cases through metadata.
+
+== Customization Layer
+
+The customization layer is an instance of an existing connector descriptor.
+When you create a connector using the customization layer, the connector is
+versioned and released separately from the base connector. Using the customization
+layer enables you to develop new versions of the base connector without duplicating code.
+Changes that are made in the new version do not affect the base version of the connector,
+nor does it break backward compatibility.
```

Comment:
I got a little confused by "base" connector and felt like I wanted to make clear that this "base" is actually the connector that you just created using the connector descriptor you just created: yes? So this is just a simple way to create a customized version of the original that is separate from the original, versus replacing the original via customization?

Reply:
I'd like to see the term "base connector" used in the Create a Connector topic, just so it maps to this one.

Reply:
It would be SWELL to have a conceptual graphic that illustrates this some time: the scaffolding and layers made visual.

Reply:
"a simple way to create a customized version of the original that is separate from the original, versus replacing the original via customization?" --- Yes!

I like that idea, added the base connector term to the create connector topic.